src/popover/popover.ts

```diff
@@ -115,6 +115,10 @@ export class NgbPopover implements OnInit, OnDestroy {
    * Currently only supports "body".
    */
   @Input() container: string;
+  /**
+   * A flag indicating if a given popover is disabled and should not be displayed.
+   */
+  @Input() disablePopover = false;
```

Comment:
You just forgot `this.disablePopover = config.disablePopover` in the constructor, and no need to initialize it here

Reply:
good catch! fixed)